bleep-core/src/scala/bleep/internal/ScriptShelloutCommand.scala

```diff
@@ -0,0 +1,18 @@
+package bleep
+package internal
+
+object ScriptShelloutCommand {
+  def getForShellScript(scriptDef: model.ScriptDef.Shell): List[String] = {
+    val bareCommand: String = scriptDef.`override-os`.flatMap(_.get(OsArch.current.os)).orElse(scriptDef.command).getOrElse {
+      throw new BleepException.Text(s"no command found for os ${OsArch.current.os}")
+    }
+    OsArch.current.os match {
+      case bleep.model.Os.Windows =>
+        List("cmd.exe", "/C", bareCommand)
+      case _ =>
+        val quote = '"'.toString
+        val quotedQuote = "\\\\"
+        List("bash", "-c", bareCommand.replace(quote, quotedQuote))
```

Comment:
not sure about this replacement. notable the `quotedQuote` is missing a quote. maybe just write script to temporary file instead?

Reply:
Tbh this was copied verbatim from [there](https://github.com/oyvindberg/bleep/pull/247/files/d138225085c2e4973711e1ef0b614802e15dff28#diff-0159507d8e14da3d810ed850041a4895e25f75d74cb4bec8fba01265fb904a8bL33-L39), so I can't comment about why the replacement is there :/

Reply:
Right. Also no idea :D gonna have to test it then